src/dataviews/histogram-dataview-model.js

```diff
@@ -18,17 +18,18 @@ module.exports = DataviewModelBase.extend({
     if (this.get('column_type')) {
       params.push('column_type=' + this.get('column_type'));
     }
-    if (_.isNumber(this.get('start'))) {
-      params.push('start=' + this.get('start'));
-    }
-    if (_.isNumber(this.get('end'))) {
-      params.push('end=' + this.get('end'));
-    }
-    if (_.isNumber(this.get('bins'))) {
-      params.push('bins=' + this.get('bins'));
-    }
     if (_.isNumber(this.get('own_filter'))) {
```

Comment:
I don't exactly get this, own_filter and setting up bins (or even start and end) could make sense from the dataview point of view